packages/grid/_modules_/grid/hooks/features/density/densityState.ts

```diff
@@ -0,0 +1,15 @@
+import { DEFAULT_GRID_OPTIONS, Density } from '../../../models/gridOptions';
+
+export interface DensityState {
+  size: Density;
+  rowHeight: number;
```

Comment:
we have 2 sources of rowHeight now, it's a bit confusing 🤔

Reply:
Yes I also don't like it much but the problem is 2 fold. One is the actual prop `rowHeight` which is set by devs and we should not control that one. The problem is that we do need a way to change its value in order to adjust the density. The idea behind this approach is that we never touch the props but rather if we need to do something with them we change their representation in the state.